### DISCUSSION TITLE
Spline support

### DISCUSSION THREAD

**Discussion by ndftz** [uxbox/uxbox#686] [Ideas]
PenPot has the potential to support amazing spline types that bake back to standard SVG 

https://codepen.io/osublake/pen/wKGvVG

https://github.com/raphlinus/spline-research

Most javascript interpretations of splines are MIT licensed which is compatible and definitely would attract typographers, illustrators and more to PenPot.

**Comment by elhombretecla** [comment]
Thank you very much for your feedback!

We will review and evaluate your proposal with the core team :)
